clang/test/CodeGenSYCL/device_has.cpp

```diff
@@ -8,25 +8,25 @@ queue q;
 
 // CHECK: define dso_local spir_kernel void @{{.*}}kernel_name_1{{.*}} !sycl_declared_aspects ![[ASPECTS1:[0-9]+]] !srcloc ![[SRCLOC1:[0-9]+]]
 
-// CHECK: define dso_local spir_func void @{{.*}}func1{{.*}} !sycl_declared_aspects ![[ASPECTS1]] !srcloc ![[SRCLOC2:[0-9]+]] {
+// CHECK: define dso_local spir_func void @{{.*}}func1{{.*}} !sycl_declared_aspects ![[ASPECTS1]] !srcloc ![[SRCLOC2:[0-9]+]]
```

Comment:
I'm trying to understand why the clang-only test was changed :)

Reply:
Hm, I fixed this test because it was failed. Seems like we are.

Reply:
https://github.com/intel/llvm/blob/sycl/clang/lib/CodeGen/BackendUtil.cpp#L880

Reply:
I assume `sycl_used_aspects ` metadata being added here since the pass runs even if we pass -disable-llvm-passes. Can you modify the CHECK to test for it as well?

Reply:
Exactly. Sure thing

Reply:
Done [435c611](https://github.com/intel/llvm/pull/7419/commits/435c611240986b9f36aa15651a3b8b997038a2bd)

Reply:
Please modify the comment at the top of the test to indicate we check sycl_used_aspects here as well.

Reply:
@premanandrao Done [586281c](https://github.com/intel/llvm/pull/7419/commits/586281c43ceccbe4bb21c09437de47228035a6d3)